__mocks__/v8-profiler-lambda.js

```diff
@@ -19,16 +29,29 @@ function stopProfiling() {
   return {
     export: cb => {
```

Comment:
Now that you're using `export()` vs `export(callback)`, you don't need the cb argument. I think this should've been caught by eslint, so maybe (separate issue) we should make sure that's set up